### ISSUE TITLE
about SegResNet

### ISSUE THREAD

**Issue by hhloverr** [yassouali/pytorch_segmentation#36]
In line 181 in segnet.py, I think it may be `encoder[0].in_channels = in_channels`  or `encoder[0] = nn.Conv2d(in_channels, 64, kernel_size=3, stride=1, padding=1)`

**Comment by yassouali** [comment]
Yes, you are correct, thanks for the pointing the error.
